Formula/kubevious.rb

```diff
@@ -0,0 +1,105 @@
+require "language/node"
+
+class Kubevious < Formula
+  desc "Detects and prevents Kubernetes misconfigurations and violations"
+  homepage "https://github.com/kubevious/kubevious"
+  url "https://registry.npmjs.org/kubevious/-/kubevious-1.0.25.tgz"
+  sha256 "d60266ed4148d8f7bd6bb6ab8133a846a7c2621cc672183cce021b5e61d21c00"
+  license "Apache-2.0"
+
+  livecheck do
+    url "https://registry.npmjs.org/kubevious/latest"
+    regex(/["']version["']:\s*?["']([^"']+)["']/i)
+  end
```

Comment:
Simply using the out of box one would be good enough, you can test out yourself using `brew livecheck kubevious`
```suggestion
```